lib/src/cupertino/cupertino_controls.dart

```diff
@@ -754,6 +757,10 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position - const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.max(skip, beginning)));
+    log("Reversing the video", name: "Video Player");
+    Future.delayed(const Duration(milliseconds: 200), () {
```

Comment:
You may want to explain why we're delaying setting the playback speed by 200 ms.

Reply:
Yes, I added it to ensure a smooth transition .